src/config/feature-flags.ts

```diff
@@ -13,16 +13,9 @@ const launchdarklyClient = LaunchDarkly.init(envVars.LAUNCHDARKLY_KEY || "", {
 
 export enum BooleanFlags {
 	MAINTENANCE_MODE = "maintenance-mode",
-	ASSOCIATE_PR_TO_ISSUES_IN_BODY = "associate-pr-to-issues-in-body",
 	VERBOSE_LOGGING = "verbose-logging",
-	REGEX_FIX = "regex-fix",
-	USE_NEW_GITHUB_CLIENT_FOR_INSTALLATION_API = "use-new-github-client-for-installation-api",
-	RETRY_ALL_ERRORS = "retry-all-errors",
 	USE_REST_API_FOR_DISCOVERY = "use-rest-api-for-discovery",
 	SEND_PR_COMMENTS_TO_JIRA = "send-pr-comments-to-jira_zy5ib",
-	USE_REPO_ID_TRANSFORMER = "use-repo-id-transformer",
-	USE_OUTBOUND_PROXY_FOR_OUATH_ROUTER = "use-outbound-proxy-for-oauth-router",
```

Comment:
This is missing in your PR description.

Reply:
Nice catch, @gxueatlassian !
<img width="500" alt="image" src="https://user-images.githubusercontent.com/20631664/218592813-d612ef61-9d53-4edf-83f3-8523c5b70c33.png">